### ISSUE TITLE
loss computation in splitcross.py is false

### ISSUE THREAD

**Issue by adjidieng** [salesforce/awd-lstm-lm#82]
Hi,

Your code for adaptive softmax in splitcross.py is false. 

(1) For the tail, you are taking log_softmax twice and you are accounting for p(tombstone) twice. You are getting the entropy both in self.log_prob() and inside the forward function. 

(2) The parameters for the tombstones are created inside splitcross.py and are not being trained because the optimizer only takes model.parameters() into account. This will affect results and quality of generated samples.

Thanks.
